src/app/technical-record-create/technical-record-create.component.ts

```diff
@@ -7,10 +14,37 @@ import { Component, OnInit, ChangeDetectionStrategy } from '@angular/core';
   changeDetection: ChangeDetectionStrategy.OnPush
 })
 export class TechnicalRecordCreateComponent implements OnInit {
+  vehicleTypes = ['PSV', 'HGV', 'Trailer'];
+  createTechRecordForm: FormGroup;
 
-  constructor() { }
+  constructor(private _store: Store<IAppState>) {
+  }
 
   ngOnInit() {
+    this.createTechRecordForm = new FormGroup({
+      'vehicleType': new FormControl('PSV', Validators.required),
```

Comment:
we can make a interface called `CreateTechRecordVM` which has the following attribute:
```
interface CreateTechRecordVM{
vin: string,
vrm: string,
vType:string
}
```
This can then be used in the store dispatch and in the action and reducer